ooiui/templates/science/streams.html

```diff
@@ -48,6 +48,11 @@
                 <div id="streamPanel"class="panel">
                     <div class="panel-heading">
                         <div class="pagination pagination-sm pagination-right pagination-boxes"></div>
+                        <div class="color-index">
+                            <div class="twelve-hours"></div><b> 12 Hours </b>
+                            <div class="twenty-four-hours"></div><b> 24 Hours </b>
+                            <div class="older"></div><b> Older than 24 hours </b>
+                        </div>
```

Comment:
I thought you were putting a legend in here for what each color means.  Or did i miss that?